queue_services/entity-emailer/src/entity_emailer/email_processors/__init__.py

```diff
@@ -94,6 +94,7 @@ def substitute_template_parts(template_code: str) -> str:
     """
     template_parts = [
         'business-dashboard-link',
+        'business-dashboard-link-alt',
```

Comment:
Haha, I thought "alt" meant "alternate". I just figured out it means "alteration"!

Is there another short version that conveys "alteration"? Alter? Altn? Thoughts?

Reply:
Yes, that is ambiguous. I saw another file with 'ALT' for 'alteration' too. I can use 'alteration' to be more explicit if you think it is better.

Reply:
It's only when I saw the other file with ALT that I understood what it meant :)

I think consistency is important, and I don't want you to change all the other places ALT is used, so I guess just leave it. (Does anyone else feel strongly about this, one way or the other?)